### DISCUSSION TITLE
Use `nucleo` for fuzzy finding?

### DISCUSSION THREAD

**Discussion by TornaxO7** [BurntSushi/ripgrep#2650] [General] [answered]
What do you think about to use the [`nucleo`] library which seems to be *very* fast?

[`nucleo`]: https://github.com/helix-editor/nucleo

**Comment by BurntSushi** [comment, marked as answer]
Sorry I missed this question. I'm unsure of what it's asking though. The title of this question seems to be a request to add fuzzy matching to ripgrep, but the body of the question seems to ask my thoughts about nucleo specifically.

From what I can tell, "fuzzy finding" involves ranking. ripgrep doesn't do ranking. It only does exhaustive search.

Typically, ripgrep is used _with_ fuzzy finders.

I'm happy to say more, but please ask a more specific or narrow question.
